src/components/formControlscomponents/Button/Button.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+function Button({ styling, btnName, handleSend, display }) {
```

Comment:
Same as here, let's define props types for the props.

[Post to read](https://dev.to/cesareferrari/what-is-proptypes-and-how-to-use-it-in-react-2eml)

I recommend you to rename `styling` to `className`